CHANGELOG.rst

```diff
@@ -2,6 +2,16 @@
 Changelog
 =========
 
+0.8.0 (2023-06-21)
+==================
+
+Feature release
+
+* Added configurable Content-Type header for DigiD SAML
+* Implemented dynamic Level Of Assurance (LOA), using the global configuration defaults
```

Comment:
```suggestion
* Implemented a way to override the global configuration defaults for Level Of Assurance (LOA)
```

Reply:
Oh and should we mention the removal of the metadata generation functions from `views.eherkenning`?